pkgpanda/__init__.py

```diff
@@ -442,7 +447,14 @@ def validate_username(username):
         if not re.match(username_regex, username):
             raise ValidationError("Username must begin with `dcos_` and only have a-z and underscore after that")
 
-    def add_user(self, username):
+    @staticmethod
+    def validate_group(group):
+        try:
+            grp.getgrnam(group)
```

Comment:
Also note, this is currently only being called during package building: https://github.com/dcos/dcos/pull/318/files#diff-41557cabf54d8bfdb900c14c8a85e435R868

That happens on entirely different machines than we actually run the items on (and that machine does not require a journalctl group to exist).

The validation during build should be just "does this group name seem reasonable", than at package activation time we check "does the group exist, does the user belong to the group"